composer.json

```diff
@@ -23,13 +23,14 @@
         "behat/behat": "^3.7"
     },
     "require-dev": {
-        "behat/mink-extension": "^2.2",
-        "behat/mink-goutte-driver": "^1.2",
         "dealerdirect/phpcodesniffer-composer-installer": "^1.0",
         "squizlabs/php_codesniffer": "3.*",
         "escapestudios/symfony2-coding-standard": "~3.0",
         "phpmd/phpmd": "^2.15",
-        "phpstan/phpstan": "^1.10"
+        "phpstan/phpstan": "^1.10",
```

Comment:
@tannguyen04 could you please sort alphabetically

Reply:
Hi @AlexSkrypnyk 
I sorted alphabetically for composer packages.